src/main/java/io/cdap/plugin/http/common/BaseHttpConfig.java

```diff
@@ -210,9 +227,24 @@ public String getOAuth2Enabled() {
 
     @Nullable
     public String getAuthUrl() {
-        return authUrl;
+      return authUrl;
     }
 
+  @Nullable
```

Comment:
This should not be nullable, in case of null grant type it should be refresh token by default